src/rules/shorthand-property-no-redundant-values/index.js

```diff
@@ -1,7 +1,8 @@
 import valueParser from "postcss-value-parser"
 import { vendor } from "postcss"
 import {
-  cssWordIsVariable,
+  isStandardDeclaration,
+  propertyIsCustom,
```

Comment:
Seems like the rest of these functions follow an `is*` convention. Should we rename this to do the same? `isCustomProperty`?